@AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_draw.sqf

```diff
@@ -220,6 +227,12 @@ if (cameraView == "GUNNER" && player == gunner _heli) then {
         ((uiNameSpace getVariable "fza_ah64_laseit") displayCtrl 701) ctrlSetText "\fza_ah64_US\tex\HDU\Apache_LaserOn.paa";
         ((uiNameSpace getVariable "fza_ah64_laseit") displayCtrl 701) ctrlSetTextColor[(_hduColour select 1), (_hduColour select 1), (_hduColour select 1), 1];
     };
+    //LSC SYMBOLOGY FOR GUNNER
+    if (_heli getvariable "fza_ah64_LmcActive") then {
+        _LMCLayer cutrsc["fza_ah64_LMC", "PLAIN", 0.01, false];
+        ((uiNameSpace getVariable "fza_ah64_LMC") displayCtrl 703) ctrlSetText "\fza_ah64_US\tex\HDU\TADSLMC_co.paa";
```

Comment:
Does this symbology really need its own RSC layer? Can it not be integrated with the rest?

Reply:
moved